<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-01-25T09:49:25 IST" name="Default suite" finished-at="2024-01-25T09:49:25 IST" duration-ms="122">
    <groups>
      <group name="regression">
        <method signature="PracticeTest.practiceOne()[pri:0, instance:practice.PracticeTest@4d5d943d]" name="practiceOne" class="practice.PracticeTest"/>
        <method signature="PracticeTest.practiceThree()[pri:0, instance:practice.PracticeTest@4d5d943d]" name="practiceThree" class="practice.PracticeTest"/>
        <method signature="PracticeTest.practiceTwo()[pri:0, instance:practice.PracticeTest@4d5d943d]" name="practiceTwo" class="practice.PracticeTest"/>
      </group> <!-- regression -->
      <group name="smoke">
        <method signature="PracticeTest.practiceOne()[pri:0, instance:practice.PracticeTest@4d5d943d]" name="practiceOne" class="practice.PracticeTest"/>
      </group> <!-- smoke -->
    </groups>
    <test started-at="2024-01-25T09:49:25 IST" name="Default test" finished-at="2024-01-25T09:49:25 IST" duration-ms="122">
      <class name="practice.PracticeTest">
        <test-method signature="practiceOne()[pri:0, instance:practice.PracticeTest@4d5d943d]" started-at="2024-01-25T09:49:25 IST" name="practiceOne" finished-at="2024-01-25T09:49:25 IST" duration-ms="12" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- practiceOne -->
        <test-method signature="practiceThree()[pri:0, instance:practice.PracticeTest@4d5d943d]" started-at="2024-01-25T09:49:25 IST" name="practiceThree" finished-at="2024-01-25T09:49:25 IST" duration-ms="9" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.fail(Assert.java:116)
at practice.PracticeTest.practiceThree(PracticeTest.java:18)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- practiceThree -->
        <test-method signature="practiceTwo()[pri:0, instance:practice.PracticeTest@4d5d943d]" started-at="2024-01-25T09:49:25 IST" name="practiceTwo" finished-at="2024-01-25T09:49:25 IST" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- practiceTwo -->
      </class> <!-- practice.PracticeTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
